lib/new_relic/agent/instrumentation/action_mailbox_subscriber.rb

```diff
@@ -0,0 +1,66 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'new_relic/agent/instrumentation/notifications_subscriber'
+
+module NewRelic
+  module Agent
+    module Instrumentation
+      class ActionMailboxSubscriber < NotificationsSubscriber
+        def start(name, id, payload)
+          return unless state.is_execution_traced?
+
+          start_segment(name, id, payload)
+        rescue => e
+          log_notification_error(e, name, 'start')
+        end
+
+        def finish(name, id, payload)
+          return unless state.is_execution_traced?
+
+          finish_segment(id, payload)
+        rescue => e
+          log_notification_error(e, name, 'finish')
+        end
+
+        def start_segment(name, id, payload)
+          segment = Tracer.start_segment(name: metric_name(name, payload))
+          segment.params[:key] = payload[:key]
+          segment.params[:exist] = payload[:exist] if payload.key?(:exist)
```

Comment:
I think this might've been a copy/paste mistake. I think you just want to include `:mailbox`? Or maybe no params since it's in the name?

Reply:
Ah, I started using the `:mailbox` payload key/val pair in `#metric_name` but then forgot to come back here and remove these 2 copy/paste keys. Thanks for catching that! I've gone ahead and gone with the "no params" approach. The only other param besides the mailbox class name that we're using in the segment name is an email address which I'm deliberately not going to store.

Addressed with [e71e37b](https://github.com/newrelic/newrelic-ruby-agent/pull/1740/commits/e71e37bde75ea6622e789faeecf8c8976a66dd53)